Resources/Prototypes/_Goobstation/_Pirates/faction.yml

```
@@ -0,0 +1,15 @@
+  - NanoTrasen
+  - Syndicate
+  - SimpleHostile
+  - Xeno
+  - PetsNT
+  - Zombie
+  - Dragon
+  - AllHostile
+  - Changeling # goob edit - changelings
+  - Heretic # goob edit - heretics
+  - Blob # Goobstation - Blob
+  - Wizard # goob edit - wizard
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)